### ISSUE TITLE
[Discovery] Extend DiscoveryV1.updateDocument to match DiscoveryV1.addJsonDocument

### ISSUE THREAD

**Issue by GwilymNewton** [watson-developer-cloud/node-sdk#477]
This is an extension of #474 

The conclusion of which was to add  a addJsonDocument function to the Discovery SDK, Which is working great, but now I need to update documents. So I would request a updateJsonDocument also be added with roughly the same functionality change. To work as below

```var document_obj = {
  document_id = id,
  environment_id: environment,
  collection_id: collection,
  file: msg.payload
};

discovery.updateJsonDocument(document_obj, function (err, response) {
  if (err) {
    console.error(err);
  } else {
    console.log(JSON.stringify(response, null, 2));
  }
});```

**Comment by nfriedly** [comment]
Oh, yea, good point. Can you send me a PR with the addition?

**Comment by GwilymNewton** [comment]
@nfriedly I thought it would be faster for you to write, as I guess it a C&P job, I just dont know which bit of your code to C&P. If not I can have a dig into the code

**Comment by GwilymNewton** [comment]
@nfriedly Added a pull request.

**Comment by nfriedly** [comment]
Fixed in #481, released in `v2.35.0`
